### PR TITLE
Standard formatting

### PR DESCRIPTION
**Pull request by khaosdoctor**
Did some corrections here

- Corrected some texts in the readme
- Added TOC to readme
- Corrected examples on readme

I also formatted the whole code to the standard formatting we've been using

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,79 +1,89 @@
 # Optimus
 
-This package is designed to receive to apply a json pipeline tratments on a image.
+> Image transformation microservice
+
+## Summary
+
+- [Optimus-prime](#optimus-prime)
```
Comment (italojs) [inline Readme.md]: the name's project is "optimius"

File: Readme.md [hunk 2]
```diff
@@ -1,79 +1,89 @@
...
-## How it's works?
 ![](./docs/images/diagram.png)
 
 ## Getting start
 
 ### Setup the optimus
 
 1º Clone this repository
-2º load the environment variables, you can finde the required envirement variables in .env.sample 
+2º load the environment variables, you can finde the required envirement variables in .env.sample
```
Comment (italojs) [inline Readme.md]: finde -> find

Comment (khaosdoctor) [reply Readme.md]: Fixed!

File: Readme.md (hunk 1 above)
Comment (khaosdoctor) [reply Readme.md]: Fixed

**Pull request merged**